I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 201, in script call
    call choices
  File "game/script.rpy", line 201, in script call
    call choices
ScriptError: could not find label 'choices'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 201, in script call
    call choices
  File "C:\Users\elhlyn11_ecolecathol\Downloads\renpy-8.3.2-sdk\renpy\ast.py", line 1414, in execute
    rv = renpy.game.context().call(label, return_site=self.next.name)
  File "game/script.rpy", line 201, in script call
    call choices
  File "C:\Users\elhlyn11_ecolecathol\Downloads\renpy-8.3.2-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'choices'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.3.2.24090902
Lovelorn Reality 1.0
Thu Jan  9 13:05:15 2025
